app/models/order.rb

```diff
@@ -0,0 +1,13 @@
+class Order < ActiveRecord::Base
+  belongs_to :user
+  has_many :order_staches
+  has_many :staches, through: :order_staches
+
+  def total
+    staches.map(&:subtotal_num).sum
+  end
+
+  def completed?
+    %w(completed cancelled).include?(status) ? "Yes" : "No"
```

Comment:
I would be fine with this change, I thought in the view it would be easier to read if it was like this.